spec/Appendix A -- Persisted Documents.md

```diff
@@ -0,0 +1,269 @@
+# A. Appendix: Persisted Documents
+
+This appendix defines an optional extension to the GraphQL-over-HTTP protocol
+that allows for the usage of "persisted documents".
+
+:: A _persisted document_ is a GraphQL document (strictly: an
+[`ExecutableDocument`](https://spec.graphql.org/draft/#ExecutableDocument)) that
+has been persisted such that the server may retrieve it based on an identifier
+indicated in the HTTP request.
+
+This feature can be used as an operation allow-list, as a way of improving the
```

Comment:
Should we go for enabling, I know that HTTP Get and URL encoding the query-string enables the same thing but it's limiting while this just blanket enables you to do this

```suggestion
This feature can be used as an operation allow-list, as a way of enabling the
```

Reply:
Out of an abundance of caution I think we should avoid this change to prevent it being used as evidence that "GraphQL doesn't do caching" without X, Y and Z. It's a nice edit though. Perhaps we could soften it, `as a simple way of setting up caching of GraphQL operations` or something...